expyfun/stimuli/_hrtf.py

```diff
@@ -40,6 +40,19 @@ def _get_hrtf(angle, source, fs):
     0-degree BRIRs (mean of that across channels) is equal to 1. This will
     ensure that the RMS of a white signal filtered with this signal is
     unchanged.
+    
+    If interp is True, the function takes the two nearest known HRTFs (in the 
+    form of a BRIR) and uses them to calculate an HRTF (again, as a BRIR) 
```

Comment:
Techincally, you're doing it in the frequency domain, so some of these caveats about BRIR are probably not necessary.